packages/redux-query/src/middleware/query.js

```diff
@@ -33,51 +60,99 @@ const defaultConfig = {
     statusCodes.SERVICE_UNAVAILABLE,
     statusCodes.GATEWAY_TIMEOUT,
   ],
-  getQueryKey,
 };
 
-const getPendingQueries = queries => {
-  return pickBy(queries, query => query.isPending);
+const getPendingQueries = (queries: QueriesState): QueriesState => {
+  const pendingQueries = {};
+
+  for (const queryKey in queries) {
+    if (queries.hasOwnProperty(queryKey)) {
+      const query = queries[queryKey];
+
+      if (query.isPending) {
+        pendingQueries[queryKey] = query;
+      }
+    }
+  }
+
+  return pendingQueries;
+};
+
+const pick = (source: { [key: string]: any }, keys: Array<string>): { [key: string]: any } => {
+  const picked = {};
+
+  for (const key of keys) {
+    if (source.hasOwnProperty(key)) {
+      picked[key] = source[key];
+    }
+  }
+
+  return picked;
 };
 
-const isStatusOK = status => status >= 200 && status < 300;
+const isStatusOk = (status: ?Status): boolean =>
+  status !== null && status !== undefined && status >= 200 && status < 300;
```

Comment:
`undefined` check is not necessary here, as `undefined < 200 === false && undefined > 300 === false`

Reply:
Flow is picky here and I prefer the explicitness. Although reading it now I think I'd prefer to write it like `typeof status === 'number' &&`